tests/defender/system/aws_iot_tests_defender_system.c

```diff
@@ -168,17 +174,8 @@ TEST_SETUP( Defender_System )
     _serverInfo = ( IotNetworkServerInfo_t ) IOT_TEST_NETWORK_SERVER_INFO_INITIALIZER;
 
     /* Set fields of start info. */
-    _startInfo.mqttNetworkInfo = ( IotMqttNetworkInfo_t ) IOT_MQTT_NETWORK_INFO_INITIALIZER;
-    _startInfo.mqttNetworkInfo.createNetworkConnection = true;
-    _startInfo.mqttNetworkInfo.u.setup.pNetworkServerInfo = &_serverInfo;
-    _startInfo.mqttNetworkInfo.u.setup.pNetworkCredentialInfo = &_credential;
-
-    _startInfo.mqttNetworkInfo.pNetworkInterface = IotNetworkMetrics_GetInterface();
-
-    _startInfo.mqttConnectionInfo = ( IotMqttConnectInfo_t ) IOT_MQTT_CONNECT_INFO_INITIALIZER;
-    _startInfo.mqttConnectionInfo.pClientIdentifier = AWS_IOT_TEST_SHADOW_THING_NAME;
-    _startInfo.mqttConnectionInfo.clientIdentifierLength = ( uint16_t ) strlen( AWS_IOT_TEST_SHADOW_THING_NAME );
-
+    _startInfo.pClientIdentifier = AWS_IOT_TEST_SHADOW_THING_NAME;
```

Comment:
`SHADOW_THING_NAME` in a defender test (I know that it does not affect the tests!)? Can you define a `DEFENDER` thing name to avoid any confusion?